src/main/java/com/example/bookstoreapp/utils/IdGenerator.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bookstoreapp.utils;
+
+import java.util.UUID;
+
```

Comment:
Used to give the unique id , but this is not a good approach in a distributed system as it is based on timestamp.

https://chronicle.software/unique-timestamp-identifiers/